src/App.js

```diff
@@ -1,22 +1,27 @@
 import { useState } from "react";
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
This dependency is not being used. Also, we don't necessarily need to use React Router. The only benefit would be the user be able to type the "edit page" URL in the browser and go directly to the edit page.

Reply:
alright. i used Link from react-router-dom on ProductList.js. For navigation without page reload. if there are other options - i'll revert this for now and we can review routing later.